applications/autoencoders/data/pilot1/data_reader_candle_pilot1.prototext

```diff
@@ -0,0 +1,25 @@
+data_reader {
+  reader {
+    name: "csv"
```

Comment:
perhaps "P1B3" would be a more appropriate name?

Reply:
I think this is right. The fields in the data reader don't have the best names. A better name would be "type", so having a CSV data reader is what we want.

Reply:
oh, that is confusing - especially because other application examples don't use that field as "type": https://github.com/LLNL/lbann/blob/develop/applications/vision/data/mnist/data_reader.prototext